source/Utils/include/FilterClusters.hxx

```diff
@@ -0,0 +1,57 @@
+#pragma once
+
+#include <marlin/Processor.h>
+
+//#include <ACTSTracking/GeometryIdMappingTool.hxx>
+#include "/opt/ilcsoft/muonc/ACTSTracking/v1.1.0/ACTSTracking/GeometryIdMappingTool.hxx"
```

Comment:
This actually should be reverted to what is in the line above, i.e.:
```c++
#include <ACTSTracking/GeometryIdMappingTool.hxx>
```
however this had problems because the ACTSTracking header files were not installed. If that is fixed this can be changed as well.